rotkehlchen/db/history_events.py

```diff
@@ -503,7 +505,12 @@ def get_history_events(
                     data = entry[data_start_idx:HISTORY_BASE_ENTRY_LENGTH + 1]
                     deserialized_event = HistoryEvent.deserialize_from_db(entry[data_start_idx:])
             except (DeserializationError, UnknownAsset) as e:
-                log.debug(f'Failed to deserialize history event {entry} due to {e!s}')
+                error = f'Failed to deserialize history event {entry} due to {e!s}'
+                log.error(error)
+                if msg_aggregator is not None:
```

Comment:
As you can see the entry is in the log but not in the message. It is useful there because we would ask the user for the logs but is not useful for the user to have the history event tuple. I would tell the user the identifier of the event and then if the reason for the error is `UnknownAsset` you can show the try redecoding the event. If it is a deserialization error I believe is not useful to re-decode and they should contact us